source/Octopus.Manager.Tentacle.Tests/WizardFixtures/SetupTentacleWizardFixture.cs

```diff
@@ -70,6 +70,61 @@ public void WhenSettingUpPollingTentacle_ScriptShouldBeCorrectlyGenerated()
             script.Should().Be(expectedOutput);
         }
         
+        [Test]
+        public void WhenSettingUpPollingTentacle_AndServerRegistrationIsSkipped_ScriptShouldBeCorrectlyGenerated()
```

Comment:
I don't think skipping the registration is supported.

The `TentacleActiveDetailsTab` control does not enable the skip button, which means that the `SkipServerRegistration` property will never be set to `true` as [this happens as part of the `OnSkip` handler](https://github.com/OctopusDeploy/OctopusTentacle/blob/964282f313065081392018917f44d0fe75343897/source/Octopus.Manager.Tentacle/TentacleConfiguration/SetupWizard/Views/TentacleActiveDetailsTab.xaml.cs#L32).

Rather than add a test for functionality that has been deprecated, I think it would be better to add a TODO to remove the skip server registration functionality.

Adding a test implies that this functionality is important and should work, when in fact it just hasn't been cleaned up properly. We don't want to pay the cost of maintaining functionality and test code for a feature we never use.